src/game-support/the-sims-2.md

```diff
@@ -0,0 +1,15 @@
+# The Sims 2 (Ultimate Collection)
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+> [!WARNING]
+> The Sims 2 is reliant on DirectX 9, which is largely unsupported by Whisky at this stage
```

Comment:
A little bit pedantic, but leave a dot at the end of the phrase and it is fine.

Reply:
Updated both remarks, cheers